### PR TITLE
libpng: tune CONFLICTS_devel; cosmetic changes.

### PR DESCRIPTION
**Pull request by fbrosson**
* libpng16: CONFLICTS_devel **+=** libpng12_devel.
* libpng12: CONFLICTS_devel = libpng_devel + libpng16_devel.
* ~~libpng: CONFLICTS_devel = libpng12_devel + libpng16_devel.~~
* libpng: drop empty $manDir in the base package, libpng.

### CODE REVIEW

File: media-libs/libpng/libpng12-1.2.56.recipe [hunk 1]
```diff
@@ -11,13 +11,13 @@ COPYRIGHT="1998-2015 Glenn Randers-Pehrson
...
-REVISION="2"
+REVISION="3"
 SOURCE_URI="http://prdownloads.sourceforge.net/libpng/libpng-$portVersion.tar.gz"
 CHECKSUM_SHA256="0366f932883084a6972efa8eadfaa5584e94a7c9c8682a49590759951518801f"
 SOURCE_DIR="libpng-$portVersion"
 
-ARCHITECTURES="x86_gcc2"
-SECONDARY_ARCHITECTURES="x86_gcc2"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```
Comment (korli) [inline media-libs/libpng/libpng12-1.2.56.recipe]: We don't want to support more architectures on outdated versions.

File: media-libs/libpng/libpng-1.5.27.recipe [hunk 2]
```diff
@@ -33,6 +33,10 @@ PROVIDES_devel="
 REQUIRES_devel="
 	libpng$secondaryArchSuffix == $portVersion base
 	"
+CONFLICTS_devel="
+	libpng12${secondaryArchSuffix}_devel
+	libpng16${secondaryArchSuffix}_devel
+	"
```
Comment (korli) [inline media-libs/libpng/libpng-1.5.27.recipe]: It's not required: the conflicts are already declared on libpng1*_devel packages.

File: media-libs/libpng/libpng12-1.2.56.recipe (hunk 1 above)
Comment (fbrosson) [inline media-libs/libpng/libpng12-1.2.56.recipe]: ok

File: media-libs/libpng/libpng-1.5.27.recipe (hunk 2 above)
Comment (fbrosson) [inline media-libs/libpng/libpng-1.5.27.recipe]: ok

**Pull request merged**